diff -aur Python-3.9.0rc1/Modules/posixmodule.c Python-3.9.0rc1.patched/Modules/posixmodule.c
--- Python-3.9.0rc1/Modules/posixmodule.c	2020-08-11 21:58:17.000000000 +0300
+++ Python-3.9.0rc1.patched/Modules/posixmodule.c	2020-09-04 13:28:41.366879592 +0300
@@ -125,8 +125,9 @@
 #  undef HAVE_SCHED_SETAFFINITY
 #endif
 
-#if defined(HAVE_SYS_XATTR_H) && defined(__GLIBC__) && !defined(__FreeBSD_kernel__) && !defined(__GNU__)
+#if defined(HAVE_SYS_XATTR_H) && defined(__linux__) && !defined(__FreeBSD_kernel__) && !defined(__GNU__)
 #  define USE_XATTRS
+#  include <linux/limits.h>
 #endif
 
 #ifdef USE_XATTRS
